recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-kernel/linux/linux-renesas/r8a7795-salvator-x-dom0.dts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Device Tree Source for the Salvator-X board
+ * running XEN hypervisor
+ *
+ * Copyright (C) 2015 Renesas Electronics Corp.
+ *
+ * This file is licensed under the terms of the GNU General Public License
+ * version 2.  This program is licensed "as is" without any warranty of any
+ * kind, whether express or implied.
+ */
+
+#include "r8a7795-salvator-x-4x2g-dom0.dts"
+
+/ {
+	model = "Renesas Salvator-X board based on r8a7795 ES2.0+";
+
+	/delete-node/memory@48000000;
```

Comment:
Are you sure you need this line?
Below you just overwrite that node with the new value.

Reply:
will remove, was not sure if I have to remove it before overwriting